test.js

```diff
@@ -185,6 +185,25 @@ experiment('nes work as normal', async () => {
   pubSubTest()
 })
 
+experiment('with subscription filter', async () => {
+  let server
+
+  beforeEach(async () => {
+    server = await start(getServer(), null, {
+      filter: function (path, message, options) {
+        return true
+      }
+    })
```

Comment:
Can you add a test where the filter function return false? Also, you'll need to add some assertions on `path`, `message` and `options`.

Reply:
If the filter function returns false the test times out b/c the message doesn't arrive. I'm not sure how to assert this. I will add assertions for three params as soon as I got some time.

Reply:
Just create a different experiment where you do not call `pubSubTest`, but rather write actual tests for things. Also, it will require a test for promise form of the async function.